test/extensions/filters/listener/common/fuzz/listener_filter_fakes.h

```diff
@@ -0,0 +1,95 @@
+#include "common/api/os_sys_calls_impl.h"
+#include "common/network/io_socket_handle_impl.h"
+#include "common/network/utility.h"
+
+#include "test/mocks/network/mocks.h"
+
+#include "gmock/gmock.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ListenerFilters {
+
+class FakeConnectionSocket : public Network::MockConnectionSocket {
```

Comment:
Please split this into a .cc file and a .h file. Having the definitions inline can be helpful but increases compilation time for every binary that `#include`s the header since it requires parsing a bunch of redundant code.

Reply:
Fixed in the latest commit!